pkg/apis/storage/validation/validation.go

```diff
@@ -445,3 +446,24 @@ func validatePodInfoOnMount(podInfoOnMount *bool, fldPath *field.Path) field.Err
 
 	return allErrs
 }
+
+// validateMode tests if mode has one of the allowed values.
+func validateMode(mode *storage.CSIDriverMode, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	if mode != nil {
+		switch *mode {
+		case storage.PersistentDriverMode:
```

Comment:
nit: golang style is to collapse these into comma delimited 

```
case storage.PersistentDriverMode, storage.EphemeralDriverMode, ...:
default:
```

since it's more canonical